targets/unity-v2/source/Authentication/Strategies/GameCenterAuthStrategy.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using PlayFab.ClientModels;
+
+namespace PlayFab.Authentication.Strategies
+{
+    internal sealed class GameCenterAuthStrategy : IAuthenticationStrategy
```

Comment:
Why doesn't GameCenter have a CustomId silent auth fallback like email?
Basically, now I'm thinking that the silent auth backup should be an automatic call to another strategy, and it should be part of the core service to pick an appropriate silent auth backup, and just do it. (Added a comment to email to this effect)